api/src/main/java/com/strategyobject/substrateclient/api/pallet/polkadot_xcm/PolkadotXcm.java

```diff
@@ -0,0 +1,177 @@
+package com.strategyobject.substrateclient.api.pallet.polkadot_xcm;
+
+import com.strategyobject.substrateclient.pallet.annotation.Event;
+import com.strategyobject.substrateclient.pallet.annotation.Pallet;
+import com.strategyobject.substrateclient.rpc.api.primitives.Hash;
+import com.strategyobject.substrateclient.rpc.api.primitives.Hash256;
+import com.strategyobject.substrateclient.scale.ScaleType;
+import com.strategyobject.substrateclient.scale.annotation.Scale;
+import com.strategyobject.substrateclient.scale.annotation.ScaleReader;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigInteger;
+import java.util.Optional;
+
+// THIS PALLET IS NONFUNCTIONAL
+@Pallet("PolkadotXcm")
+public interface PolkadotXcm {
+
+    @Event(index = 0)
+    @Getter
+    @Setter
+    @ScaleReader
+    class Attempted {
+        @Scale(ScaleType.U32.class)
```

Comment:
How did you determine this type?

Reply:
Whoops ,used the wrong type big time on that one.